Parser/Functions/RichPresenceDisplayFunction.cs

```diff
@@ -89,15 +95,19 @@ private ParseErrorExpression ProcessRichPresenceDisplay(StringConstantExpression
                 }
                 tokenizer.Advance();
 
-                var parameterIndex = Int32.Parse(index.ToString());
-                if (parameterIndex >= varargs.Entries.Count)
+                Int32 parameterIndex;
+                if (!Int32.TryParse(index.ToString(), out parameterIndex)
+                    || parameterIndex >= varargs.Entries.Count)
```

Comment:
should probably also check for negative values

Reply:
Agreed, added negative value check.